webapp/src/features/adminEditRsu/AdminEditRsu.tsx

```diff
@@ -361,6 +372,11 @@ const AdminEditRsu = () => {
             </button>
           </div>
         </Form>
+      ) : (
+        <Typography variant={'h4'} style={{ color: '#fff' }}>
+          Unknown RSU IP address. Either this RSU does not exist, or you do not have access to it.{' '}
+          <Link to="dashboard/admin/rsus">RSUs</Link>
```

Comment:
Clicking this link gets us to an error page: 
![image](https://github.com/CDOT-CV/jpo-cvmanager/assets/2932159/7a396946-31fb-47c6-8ca5-9e88aeb8feb8)

Reply:
Yup, I changed the links to be absolute and didn't put the starting slash... fixed...